Makefile

```
@@ -218,7 +222,7 @@ cargo-deps:
 	cargo install --version 1.19.0 hyperfine
 	cargo install --version 0.9.49 cargo-nextest --locked
 	cargo install --version 0.5.9 cargo-llvm-cov
-	cargo install --version 0.12.1 wasm-pack --locked
+	# cargo install --version 0.12.1 wasm-pack --locked
```

Hmm, missed removing that comment.
```suggestion
	cargo install --version 0.12.1 wasm-pack --locked
```